lib/reply.js

```diff
@@ -88,10 +88,20 @@ Reply.prototype.hasHeader = function (key) {
   return this._headers[key.toLowerCase()] !== undefined
 }
 
+Reply.prototype.removeHeader = function (key) {
```

Comment:
The method could return `this` because it has no return value.

Reply:
Agree